cmd/wego/add/cmd.go

```diff
@@ -30,9 +30,18 @@ var Cmd = &cobra.Command{
 	Use:   "add [--name <name>] [--url <url>] [--branch <branch>] [--path <path within repository>] [--private-key <keyfile>] <repository directory>",
 	Short: "Add a workload repository to a wego cluster",
 	Long: strings.TrimSpace(dedent.Dedent(`
-        Associates an additional application in a git repository with a wego cluster so that its contents may be managed via GitOps
-    `)),
-	Example: "wego app add .",
+		Associates an additional application in a git repository with a wego cluster so that its contents may be managed via GitOps
+	`)),
+	Example: `
+  # Add podinfo application to wego control from local git repository
+  wego app add .
+
+  # Add podinfo application to wego control from github repository
+  wego app add github.com/myorg/podinfo
```

Comment:
I believe this needs `--url=` and either:

- `git@github.com:myorg/podinfo`
- `ssh://git@github.com/myorg/podinfo`
- `https://github.com/myorg/podinfo`